blocks/rich-text/index.js

```diff
@@ -711,9 +784,20 @@ export class RichText extends Component {
 	}
 
 	componentDidUpdate( prevProps ) {
+		if ( ! this.editor ) {
+			return;
+		}
+
+		if ( this.props.isSelected !== prevProps.isSelected ) {
+			this.toggleInsertAvailable();
+		}
+
+		if ( this.props.inlineBlock && this.props.isSelected ) {
+			this.insertInlineBlock();
```

Comment:
Hm, I don't understand the `inlineBlock` prop here. It doesn't sound like it would trigger an action.

Reply:
Good point, I renamed it to `inlineBlockForInsert` in https://github.com/WordPress/gutenberg/pull/5794/commits/d5b87664194cefe7e54b2fd1fc304eb91260d27a. If that is still not clear, please let me know. This prop gets the `inlineBlockForInsert` state from the editor store when it is added there by the media library in the inserter.